package.json

```diff
@@ -28,7 +28,8 @@
         "joi": "17.4.0",
         "spdx-expression-parse": "3.0.1",
         "spdx-satisfies": "5.0.0",
-        "tslib": "2.1.0"
+        "tslib": "2.1.0",
+        "xmlbuilder": "^15.1.1"
```

Comment:
Remove ^: "xmlbuilder": "15.1.1".
License number must be specific for Renovate.